### PR TITLE
YD-492 Guarded DB stats through test

### PR DESCRIPTION
**Pull request by Bert-R**
Created a test that guards the database access statistics of the analysis engine service. This service is very performance critical, so it is important that the database access is very limited.

The HibernateStatisticsController is now moved to Core, so each Yona service has the URL path to manage the statistics.

### CODE REVIEW

File: core/src/testUtils/groovy/nu/yona/server/test/AnalysisService.groovy [hunk 1]
```diff
@@ -40,8 +39,38 @@ class AnalysisService extends Service
 					"url":"$url"
 				}""")
 	}
+
 	def postToAnalysisEngine(String vpnLoginId, jsonString)
 	{
 		yonaServer.postJson(USER_ANONYMIZED_PATH + vpnLoginId + "/networkActivity/", jsonString)
 	}
+	void setEnableStatistics(def enable)
```
Comment (aristotelos) [inline core/src/testUtils/groovy/nu/yona/server/test/AnalysisService.groovy]: This part is duplicate code with `AppService`. Can't we merge this to `YonaServer`?

Comment (Bert-R) [reply core/src/testUtils/groovy/nu/yona/server/test/AnalysisService.groovy]: Makes sense. Moved it.

**Comment by Bert-R** [comment]
Pushed an update

**Comment by Bert-R** [comment]
Don't merge this yet. Something is wrong in my last change set.

**Comment by aristotelos** [comment]
I meant some more methods, `resetStatistics`, `clearCaches`, `getStatistics` as well. And do we need a wrapper method `setEnableStatistics` around `yonaServer.enableStatistics`.

**Comment by Bert-R** [comment]
Oops, I overlooked these.

**Comment by Bert-R** [comment]
Pushed an update.

**Pull request merged**